style.css

```diff
@@ -437,15 +443,15 @@
   }
 
   .arrow1 {
-    height: 3%;
-    width: 6%;
+    height: 1%;
+    width: 7%;
     margin-left: 160%;
     margin-top: -10%;
   }
 
   .arrow2 {
-    height: 3%;
-    width: 6%;
+    height: 1%;
+    width: 7%;
     margin-left: 160%;
     margin-top: -10%;
   }
```

Comment:
[OPTIONAL]
- [ ] Please stick to the design as much as possible (e.g., font, colors, images, text, margins).
This is a requirement of the project but the first reviewer didn't notice it. So I requested this as an optional but I highly recommend you to make the change
### Your design
![Screenshot from 2023-03-25 09-04-53](https://user-images.githubusercontent.com/91142432/227709122-f457ba0a-0ecb-4f08-88d9-ad5320f1d93a.png)

### What is suppose to look like
![Screenshot from 2023-03-25 09-33-37](https://user-images.githubusercontent.com/91142432/227709191-3942ec12-05a7-433c-a60c-363cb7f28f2c.png)